install_config/web_console_customization.adoc

```diff
@@ -265,6 +265,10 @@ oauthConfig:
 Relative paths are resolved relative to the master configuration file. You must
 restart the server after changing this configuration.
 
+Each time a user's token to {product-title} expires, the user is presented with
```

Comment:
the alwaysShowProviderSelection configuration is discussed on this page https://github.com/openshift/openshift-docs/blob/master/install_config/configuring_authentication.adoc